### DISCUSSION TITLE
Default blacklist with banking sites

### DISCUSSION THREAD

**Discussion by Stefano-Panzeri** [violentmonkey/violentmonkey#2188] [Ideas]
Hi,
some weeks ago I noticed that a scammer was publishing some scripts on Greasyfork, all having the `// @match  *://*/*` selector.
The scripts were advertised as scripts for porn sites, but they also contained a keylogger running on banking websites.
As this is a serious threath for the majority of Violentmonkey users, while the more expert ones would probably deactivate any random userscripts from running on their banking website anyways, would it be possible to fill by default the blacklist in Violentmonkey adding the banking sites?[
This list](https://github.com/olbat/ut1-blacklists/blob/master/blacklists/bank/domains) for example includes lots of sites and having it (or a similar one) loaded by default when installing Violentmonkey would save the users from this threat, and those who really want to use a script on a banking site could just remove their bank from the blacklist.
Do you think it could be a good idea?

**Comment by tophf** [comment]
The idea is good, but I don't like the list, because it gives a false sense of security while not ensuring this particular user is protected, so I'd prefer the user specifying the domain explicitly. And, to increase awareness of this, we can maybe add a visible reminder in all our UI pages including the installer to configure exclusions for sensitive sites.

**Reply by Stefano-Panzeri** [reply]
I get your point: I was in fact looking inside that list, and for example my bank (a rather big Italian bank) was not listed, hence the false security you were saying. Reminding the users to set up exclusions for banks, insurances, government sites, etc. is definitely a better idea, as you were suggesting.
